.env.template.testnet.gobi

```diff
@@ -3,8 +3,25 @@ COMPOSE_PROJECT_NAME=evmapp-testnet-gobi
 INTERNAL_NETWORK_SUBNET='10.10.50.0/24'
 SCNODE_IP_ADDRESS='10.10.50.3'
 
+ZEN_TAG=v4.1.1
+ZEN_PORT=19033
+ZEN_RPC_USER=zenuser
+ZEN_RPC_PASSWORD=password
+ZEN_RPC_PORT=8231
+ZEN_WS_PORT=8888
+ZEN_RPC_ALLOWIP_PRESET=SUBNET
+ZEN_EXTERNAL_IP=DETECT
+ZEN_OPTS="-websocket=1 -wsaddress=0.0.0.0 -wsport=8888 -logips=1 -logtimestamps=1 -testnet=1 -disabledeprecation=4.1.1 -debug=ws -debug=sc -debug=cert"
+ZEN_CUSTOM_SCRIPT=""
+ZEN_SEED_TAR_GZ_URL=""
```

Comment:
I've not deleted the variable because I think we could want to add a seed.sh script in the future. Actually I've not added it yet (and I've not added any extra volume) because we don't have a seed to provide. Also, if we provide the script we should provide instructions on how to use it. Let's keep this for future improvement.